library/eclipse-temurin

```diff
@@ -193,3 +193,41 @@ Directory: 16/jdk/windows/nanoserver-1809
 File: Dockerfile.releases.full
 Constraints: nanoserver-1809, windowsservercore-1809
 
+
+#------------------------------v17 images---------------------------------
+Tags: 17_35-jdk-focal, 17-jdk-focal, 17-focal
+SharedTags: 17_35-jdk, 17-jdk, 17, latest
+Architectures: amd64
+GitCommit: 63defc24ed1b9051b48f6dfc747761a1b88a3b42
+Directory: 17/jdk/ubuntu
+File: Dockerfile.releases.full
+
+Tags: 17_35-jdk-centos7, 17-jdk-centos7, 17-centos7
+Architectures: amd64
+GitCommit: 63defc24ed1b9051b48f6dfc747761a1b88a3b42
+Directory: 17/jdk/centos
+File: Dockerfile.releases.full
+
+Tags: 17_35-jdk-windowsservercore-1809, 17-jdk-windowsservercore-1809, 17-windowsservercore-1809
+SharedTags: 17_35-jdk-windowsservercore, 17-jdk-windowsservercore, 17-windowsservercore, 17_35-jdk, 17-jdk, 17, latest
+Architectures: windows-amd64
+GitCommit: 63defc24ed1b9051b48f6dfc747761a1b88a3b42
+Directory: 17/jdk/windows/windowsservercore-1809
+File: Dockerfile.releases.full
+Constraints: windowsservercore-1809
+
+Tags: 17_35-jdk-windowsservercore-ltsc2016, 17-jdk-windowsservercore-ltsc2016, 17-windowsservercore-ltsc2016
+SharedTags: 17_35-jdk-windowsservercore, 17-jdk-windowsservercore, 17-windowsservercore, 17_35-jdk, 17-jdk, 17, latest
+Architectures: windows-amd64
+GitCommit: 63defc24ed1b9051b48f6dfc747761a1b88a3b42
+Directory: 17/jdk/windows/windowsservercore-ltsc2016
+File: Dockerfile.releases.full
+Constraints: windowsservercore-ltsc2016
+
+Tags: 17_35-jdk-nanoserver-1809, 17-jdk-nanoserver-1809, 17-nanoserver-1809
+SharedTags: 17_35-jdk-nanoserver, 17-jdk-nanoserver, 17-nanoserver
```

Comment:
```suggestion
SharedTags: 17_35-jdk-nanoserver, 17-jdk-nanoserver, 17-nanoserver, 17_35-jdk, 17-jdk, 17, latest
```
Should this have the generic tags too?

Reply:
same as above - we cannot have generic shared tags for nanoserver as `servercore-1809` is already default